include/miscutils.h

```diff
@@ -33,6 +33,14 @@ class GzFile : public QFile {
   qint64 writeData(const char *data, qint64 maxSize) override;
 };
 
+inline Qt::CaseSensitivity osCaseSensitivity() {
+#if defined Q_OS_WIN
```

Comment:
MacOS file systems are by default case insensitive. Please change that such that Linux is case sensitive and the rest not.